src/plugins/base-severities.js

```diff
@@ -0,0 +1,40 @@
+
+function plugin({ severityRegistry, Logger, Severity }) {
+    severityRegistry
+        .register(0, 'EMERGENCY')
+        .register(1, 'ALERT')
+        .register(2, 'CRITICAL')
+        .register(3, 'ERROR')
+        .register(4, 'WARNING')
+        .register(5, 'NOTICE')
+        .register(6, 'INFO')
+        .register(7, 'DEBUG')
+        .register(7, 'DEFAULT')
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/loggin-js/loggin-js/pullRequest?prid=7629800)
```suggestion
        .register(7, "DEFAULT")
```